ArduCopter/crash_check.cpp

```diff
@@ -65,7 +66,7 @@ void Copter::crash_check()
         // keep logging even if disarmed:
         AP::logger().set_force_log_disarmed(true);
         // send message to gcs
-        gcs().send_text(MAV_SEVERITY_EMERGENCY,"Crash: Disarming");
+        gcs().send_text(MAV_SEVERITY_EMERGENCY,"Crash: Disarming: AngErr=%.0f>%.0f, Accel=%.1f<%.1f", angle_error, CRASH_CHECK_ANGLE_DEVIATION_DEG, filtered_acc, CRASH_CHECK_ACCEL_MAX);
```

Comment:
This may need to have "(double)" placed before filtered_acc or it may create a compiler warning but let's see..

Reply:
@rmackay9 san.  Thanks for the comment. In my environment, there is no warning.